backend/experiment/rules/base.py

```diff
@@ -114,66 +109,78 @@ def rank(self, session, exclude_unfinished=True):
-                return bucket['rank']
+            if percentile >= bucket["min_percentile"]:
+                return bucket["rank"]
 
         # Default return, in case score isn't in the buckets
-        return ranks['PLASTIC']
+        return ranks["PLASTIC"]
 
     def get_single_question(self, session, randomize=False):
         """Get a random question from each question list, in priority completion order.
 
         Participants will not continue to the next question set until they
         have completed their current one.
         """
-        questionnaire = unanswered_questions(session.participant, get_questions_from_series(session.block.questionseries_set.all()), randomize)
+        questionnaire = unanswered_questions(
+            session.participant, get_questions_from_series(session.block.questionseries_set.all()), randomize
+        )
         try:
             question = next(questionnaire)
-            return Trial(
-                title=_("Questionnaire"),
-                feedback_form=Form([question], is_skippable=question.is_skippable))
+            return Trial(title=_("Questionnaire"), feedback_form=Form([question], is_skippable=question.is_skippable))
         except StopIteration:
             return None
 
     def get_open_questions(self, session, randomize=False, cutoff_index=None) -> Union[list, None]:
-        ''' Get a list of trials for questions not yet answered by the user '''
+        """Get a list of trials for questions not yet answered by the user"""
 
         trials = []
-        questions = list(unanswered_questions(session.participant, get_questions_from_series(session.block.questionseries_set.all()), randomize, cutoff_index))
+        questions = list(
+            unanswered_questions(
+                session.participant,
+                get_questions_from_series(session.block.questionseries_set.all()),
+                randomize,
+                cutoff_index,
+            )
+        )
         open_questions = len(questions)
         if not open_questions:
             return None
         for index, question in enumerate(questions):
-            trials.append(Trial(
-                title=_("Questionnaire %(index)i / %(total)i") % {'index': index+1, 'total': open_questions},
-                feedback_form=Form([question], is_skippable=question.is_skippable)
-            ))
+            trials.append(
+                Trial(
+                    title=_("Questionnaire %(index)i / %(total)i") % {"index": index + 1, "total": open_questions},
+                    feedback_form=Form([question], is_skippable=question.is_skippable),
+                )
+            )
         return trials
 
     def social_media_info(self, block, score):
-        ''' ⚠️ Deprecated. The social media info will eventually be present on the Experiment level, not the Block level. '''
+        """⚠️ Deprecated. The social media info will eventually be present on the Experiment level, not the Block level."""
```

Comment:
Can this docstring go now?

Reply:
I still see this method being called from `hooked`, `matching_pairs` and `thats_my_song` though 🤔 Are you sure it's safe?

![image](https://github.com/user-attachments/assets/29524066-791f-4e89-8e89-b87001125231)

Reply:
As discussed, in the meeting of september 3rd, we cannot remove this _yet_ as we first have to decide on how to deal with social media translations and then implement them.